packages/mini-browser/src/node/mini-browser-endpoint.ts

```diff
@@ -198,8 +198,8 @@ const CODE_EDITOR_PRIORITY = 100;
 @injectable()
 export class HtmlHandler implements MiniBrowserEndpointHandler {
 
-    supportedExtensions(): string {
-        return 'html';
+    supportedExtensions(): string[] {
+        return ['html', 'xhtml'];
```

Comment:
Should we also support `.htm` while we're here? (Haven't seen these in ages, but maybe it doesn't hurt to have it.)

```suggestion
        return ['html', 'xhtml', 'htm'];
```

Reply:
I updated the code to include `.html`, `.htm`, and `.xhtml` file types :)